OpenRA.Mods.Common/Traits/Interactable.cs

```diff
@@ -21,13 +21,13 @@ public class InteractableInfo : TraitInfo, IMouseBoundsInfo
 	{
 		[Desc("Defines a custom rectangle for mouse interaction with the actor.",
 			"If null, the engine will guess an appropriate size based on the With*Body trait.",
-			"The first two numbers define the width and height of the rectangle.",
+			"The first two numbers define the width and height of the rectangle as a world distance.",
 			"The (optional) second two numbers define an x and y offset from the actor center.")]
-		public readonly int[] Bounds = null;
+		public readonly WDist[] Bounds = null;
```

Comment:
Could it be an idea to have this as a explicit type, with the properties, width, height and optional offsets?

Reply:
I would be more keen on this if we had a way of dynamically registering types with FieldLoader. Right now we basically have to hardcode loadable types in `OpenRA.Game`*, which heavily discourages the use of any trait-specific types.

*We do have the `TypeConverter` stuff, but the plumbing overhead to implement this is larger than the types that would be using it.